rust/agama-utils/src/api/action.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) [2025] SUSE LLC
+//
+// All Rights Reserved.
+//
+// This program is free software; you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation; either version 2 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, contact SUSE LLC.
+//
+// To contact SUSE LLC about this file by physical or electronic mail, you may
+// find current contact information at www.suse.com.
+
+use crate::api::l10n;
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, utoipa::ToSchema)]
+pub enum Action {
+    #[serde(rename = "configureL10n")]
+    ConfigureL10n(l10n::SystemConfig),
```

Comment:
I still thing we will need explicit probe for software here e.g. when repository refresh failed.